### DISCUSSION TITLE
Error compiling for device -- compiler can't find include files

### DISCUSSION THREAD

**Discussion by CarlosMendonca** [boozook/playdate#423] [Q&A]
Hello! I'm having a problem that seems different from https://github.com/boozook/playdate/discussions/421.

Compiling for simulator works well, but in trying to compile for device, I get an issue with the compiler finding include files:
```
[playdate-sys 0.4.6] /usr/lib/arm-none-eabi/include/stdlib.h:16:10: fatal error: 'stddef.h' file not found

[playdate-sys 0.4.6] Couldn't generate bindings.: Bindgen(ClangDiagnostic("/usr/lib/arm-none-eabi/include/stdlib.h:16:10: fatal error: 'stddef.h' file not found\n"))
```

Environment is Ubuntu 22.04 and I believe I have all dependencies installed. Hello-world example on C_API compiles file with `make` and the `stddef.h` is indeed available on `/usr/lib/gcc/arm-none-eabi/10.3.1`. I'm sure this is a simple configuration problem -- any idea what could be wrong? Maybe a symlink missing somewhere? Appreciate any help here.

**Comment by Niashi24** [comment]
Not sure if this has anything to do with it, but I'm on the same Ubuntu (22.04.03) and `arm-none-eabi` (10.3.1) versions and my installation doesn't have a `stdlib.h` file anywhere.
